src/sentry/notifications/utils/__init__.py

```diff
@@ -335,17 +337,23 @@ def get_span_evidence_value(span: Optional[Dict[str, Union[str, float, None]]] =
     return render_to_string("sentry/emails/transactions.html", context)
 
 
+def get_matched_problem(event: Event) -> EventPerformanceProblem:
+    """Get the matching performance problem for a given event"""
+    matched_problem: EventPerformanceProblem = None
+    for problem in get_problems([event]):
+        if problem.problem.fingerprint == GroupHash.objects.get(group=event.group).hash:
+            matched_problem = problem
+    return matched_problem.problem
```

Comment:
This could error if we can't retrieve the problem for some reason. Looks like the code was already like this before, but we might want to come back to this later and add some more defensive code

Reply:
Yeah I can come back to it after given the crunch 👍

Reply:
jk this was breaking stuff, updated